src/pages/CreateFeedback.tsx

```diff
@@ -0,0 +1,202 @@
+    navigate('/');
+    event.preventDefault();
+  };
+
+  return (
+    <div className="create-feedback-page">
+      <nav>
+        <img src={leftArrow} alt="left arrow" />
+        <Link to="/">Go Back</Link>
+      </nav>
+      <img
+        src={newFeedback}
+        alt="purple circle with white plus sign"
+        className="add"
+      />
+      <form className="create-feedback">
+        <h1>Create New Feedback</h1>
+        <label htmlFor="title" className="create-feedback_heading">
+          <span>Feedback Title</span>
+          <span>Add a short, descriptive headline</span>
+        </label>
+        <input
+          type="text"
+          id="title"
+          name="title"
+          required
+          className="text"
+          value={title}
+          onChange={changeTitle}
+        />
+
+        <label htmlFor="category" className="create-feedback_heading">
+          <span>Category</span>
+          <span>Choose a category for your feedback</span>
+        </label>
+
+        <ul className="dropdown">
+          <div onClick={openDropDown} className={openBorder}>
+            <li
+              role="button"
+              aria-labelledby="dropdown-label"
+              id="dropdown__selected"
+              tabIndex={0}
+            >
+              {category}
+            </li>
+            <img src={isOpen ? arrowUp : arrowDown} alt="arrow down" />
+          </div>
+
+          <li
+            aria-expanded={isOpen}
+            role="list"
+            className="dropdown__list-container"
+          >
+            {isOpen === false ? null : (
+              <ul className="dropwdown_list">
+                <li className="dropdown_list-item" tabIndex={0}>
+                  <span
+                    id="Feature"
+                    className={category === 'Feature' ? 'currentCategory' : ''}
+                    onClick={changeCategory}
+                  >
+                    Feature
+                  </span>
+                </li>
+                <li className="dropdown_list-item" tabIndex={0}>
+                  <span
+                    id="Enhancement"
+                    className={
+                      category === 'Enhancement' ? 'currentCategory' : ''
+                    }
+                    onClick={changeCategory}
+                  >
+                    Enhancement
+                  </span>
+                </li>
+                <li className="dropdown_list-item" tabIndex={0}>
+                  <span
+                    id="Bug"
+                    className={category === 'Bug' ? 'currentCategory' : ''}
+                    onClick={changeCategory}
+                  >
+                    Bug
+                  </span>
+                </li>
+                <li
+                  className="dropdown_list-item"
+                  tabIndex={0}
+                  id="UI"
+                  onClick={changeCategory}
+                >
+                  <span
+                    id="UI"
```

Comment:
Not sure, do you need all of these `id`?

Reply:
In my changeCategory function I need to capture the event.target.id